internal/cmd/controller/gitops/reconciler/gitjob_test.go

```diff
@@ -1169,9 +1259,10 @@ func TestGenerateJob_EnvVars(t *testing.T) {
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
 			r := GitJobReconciler{
-				Client: fake.NewFakeClient(),
-				Image:  "test",
-				Clock:  RealClock{},
+				Client:          getTestClient(testClientSimple, []corev1.Toleration{}),
```

Comment:
nit: Is updating this line really necessary, if no checks are run on tolerations?

Reply:
Yes, it is necessary, because the test client now adds the Deployment to the schema. 
That is needed when creating the gitjob Spec